src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -228,22 +231,28 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
                 botResponse,
                 Instant.now());
         messageStore.addMessage(botMessage);
-        
+
         List<String> botMessageInformation = new ArrayList<String>();
         botMessageInformation.add(bot.getName());
         botMessageInformation.add(conversation.getTitle());
         botMessageInformation.add(botResponse);
         botMessageInformation.add(conversationId.toString());
-        Event botMessageEvent = 
+        Event botMessageEvent =
             new Event(
-                UUID.randomUUID(), 
-                "Message", 
-                botMessage.getCreationTime(), 
+                UUID.randomUUID(),
+                "Message",
+                botMessage.getCreationTime(),
                 botMessageInformation);
         eventStore.addEvent(botMessageEvent);
       }
     }
- 
+    String subscriptionIdAsAString = request.getParameter("subscriptionField");
```

Comment:
Not sure if 249 and 250 are needed if you already have conversationId. Also this code executes when a user sends a message or @mentions right, we only want it to execute when a user clicks on receive notifications.